docs/prod-maintain-secure-upgrade.md

```diff
@@ -416,9 +416,8 @@ the `knight` management command:
 
 If you need to manage the IRC, Jabber, or Zephyr mirrors, you will
 need to create API super users.  To do this, use `./manage.py knight`
-with the `--permission=api_super_user` argument.  See
-`api/integrations/irc-mirror.py` and
-`api/integrations/jabber_mirror.py` for further detail on these.
+with the `--permission=api_super_user` argument.  See the respective
+integration scripts for these mirrors for further detail on these.
```

Comment:
This is a bit awkward, in that neither has its own real documentation.  I think we might want to at least point to the right directory in `python-zulip-api` for where to find the code?